src/main/xar-resources/data/lucene/lucene.xml

```diff
@@ -348,6 +354,11 @@
                 <programlisting language="xml" xlink:href="listings/listing-30.xml"/>
 
                 <programlisting language="xml" xlink:href="listings/listing-31.xml"/>
+                
+                <para>For instance to construct your custom analyzer you might use
```

Comment:
Thanks a lot that makes it much clearer now!
What do you think about this way of phrasing it?
"To pass an array of characters to your custom analyzer see the example below:"

Reply:
I don't think it makes sense... otherwise we could end up adding lots of permutations of examples for different things. This is at the Java level, to use this you should know already what you are doing I think.

Reply:
It is more to exactly state what we will be looking at. To me, that makes it easier to generalise the concept. I did not suggest that all combinations of parameter types should be listed here. In fact now that I see the listing it is very similar to the existing example with CharArraySet. So a general paragraph "this is how you pass in Collections of values" would suffice for both cases and also cover `java.lang.String[]`.

The other listings do not even have such a description so that makes this already better. I think that the others should now also get a descriptive paragraph each but this is out of scope of this PR.